analysers/analyser_merge_poi_FR.py

```diff
@@ -0,0 +1,320 @@
+            tags = {'amenity': 'place_of_worship'},
+            generate_tags = {'historic': 'wayside shrine', 'amenity': 'place_of_worship', 'religion': 'christian', 'denomination': 'catholic'},
+        ),
+        dict(
+            #   2255 Temple protestant
+            item = 8100,
+            id = 10004,
+            level = 3,
+            title = T_('Place of worship not integrated'),
+            select = {'nature': 'Culte chrétien', 'nature_detaillee': 'Temple protestant'},
+            tags = {'amenity': 'place_of_worship'},
+            generate_tags = {'amenity': 'place_of_worship', 'religion': 'christian', 'denomination': 'protestant'},
+        ),
+        dict(
+            #     64 Eglise orthodoxe
+            item = 8100,
+            id = 10005,
+            level = 3,
+            title = T_('Place of worship not integrated'),
+            select = {'nature': 'Culte chrétien', 'nature_detaillee': 'Eglise orthodoxe'},
+            tags = {'amenity': 'place_of_worship'},
+            generate_tags = {'amenity': 'place_of_worship', 'religion': 'christian', 'denomination': 'orthodox'},
+        ),
+            #     46 Enclos paroissial
+            #     42 Eglise simultanée
+        # 54455 Enseignement primaire
+        # 53718 Espace public
+        # 38865 Mairie
+        dict(
+            item = 8110,
+            id = 10100,
+            level = 3,
+            title = T_('Townhall not integrated'),
+            select = {'nature': 'Mairie', 'nature_detaillee': ['', 'Mairie de commune associée ou déléguée', 'Mairie annexe', 'Hôtel de ville', 'Mairie d\'arrondissement']},
+            tags = {'amenity': 'townhall'},
+        ),
+        # 27275 Monument
+        dict(
+            #  13600 Monument aux morts
+            item = 8010,
+            id = 10200,
+            level = 3,
+            title = T_('Memorial not integrated'),
+            select = {'nature': 'Monument', 'nature_detaillee': 'Monument aux morts'},
+            tags = {'historic': 'memorial'},
+        ),
+            #   7111
+            #   5082 Stèle
+            #   1304 Statue
+            #    188 Mémorial
+
+        # 23183 Station d'épuration
+        dict(
+            item = 8500,
+            id = 10300,
+            level = 3,
+            title = T_('Wastewater plant not integrated'),
+            select = {'nature': 'Station d\'épuration'},
+            tags = {'man_made': 'wastewater_plant'},
+        ),
+        # 19335 Station de pompage
+        dict(
+            item = 8500,
+            id = 10400,
+            level = 3,
+            title = T_('Pumping station not integrated'),
+            select = {'nature': 'Station de pompage'},
+            tags = {'man_made': 'pumping_station'},
+        ),
+        # 14556 Poste
+        # 13338 Zone industrielle
+        # 13277 Autre équipement sportif
+        # 12908 Aire de détente
+        #  9952 Salle de spectacle ou conférence
+        #  9025 Camping
```

Comment:
Je ne croix pas qu'il y ait d'autre source nationale pour les camping donc je l'ajouterais bien : tourism=camp_site

Reply:
We already have the campsites from DataTourism
http://osmose.openstreetmap.fr/fr/errors/?item=8140
https://github.com/osm-fr/osmose-backend/blob/master/analysers/analyser_merge_tourism_FR.py#L86